xep-0393.xml

```diff
@@ -467,24 +525,67 @@
     text more difficult to read for visually impaired users.
   </p>
   <p>
-    Styled text may also be rendered poorly by screen readers.
+    Styled text may be rendered poorly by screen readers.
     When applying formatting it may be desirable to include directives to
-    exclude styling directives from being read.
+    exclude styling directives from being read, or to add markers to the final
+    output that have semantic meaning for screen readers.
+    For example, in an web based client an emphasis span might be converted to
+    an HTML &lt;emph&gt; element.
```

Comment:
The HTML element which I think you’re thinking of is ``<em/>``, not ``<emph/>``.

Reply:
Ah thanks, I don't do a lot of HTML but wanted an example people would be familiar with. Should have familiarized myself first. Fixed!